src/components/SocialCardDisplay.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {BsArrowDown} from "react-icons/bs";
+
+import {
+    GoLocation
+} from "react-icons/go"
+
+function SocialCardDisplay(props)
+{
+    
+    return (
+        <div className="cards py-3 px-3">
+            <div>
+            <h4 className="card-heading">
+                {props.cards.name}
+            {props.cards.logo}
+            </h4>
+            </div>
+            <p className="card-title">{props.cards.title}</p>
+            <p className="card-location"><GoLocation className="mr-2" />{props.cards.location}</p>
+            <div className="button-encloser text-center">
+                <button className="card-button btn btn-info rounded-circle text-center mb-2" onClick={() => props.toggle(props.index)} id={"b"+(props.index)}>
+                    <BsArrowDown className="" size="2rem" />
```

Comment:
ArrowDown button doesn't looks good. Also instead of rotate, change the icon of up and down. Icon is not centered. Use can use font-Awesome. CDN is already defined for that.